docs/reference/plugin/sitemap.md

```diff
@@ -0,0 +1,105 @@
+# sitemap
+
+<NpmBadge package="@vuepress/plugin-sitemap" />
+
+Generate sitemap for your vuepress project.
+
+## Install
+
+```bash
+npm i -D @vuepress/plugin-sitemap@next
+```
+
+## Frontmatter Options
+
+### sitemap.changefreq
+
+- Type: `'always'|'hourly'|'daily'|'weekly'|'monthly'|'yearly'|'never'`
+
+- Default: `'daily'`
+
+- Details:
+
+  Page default update frequency.
+
+### sitemap.exclude
+
+- Type: `boolean`
+
+- Default: `false`
+
+- Details:
+
+  Whether exclude the page from sitemap.
+
+### sitemap.priority
+
+- Type: `number`
+
+- Default: `0.5`
+
+- Details:
+
+  Page priority, range from `0` to `1`.
+
+## Plugin options
+
+### hostname
+
+- Type: `string`
+
+- Required: true
+
+- Details:
+
+  The domain name where the current site is deployed, the plugin needs this option to work.
+
+### urls
+
+- Type: `string[]`
+
+- Default: `[]`
+
+- Details:
+
+  Extra links to be included.
+
+### exclude
+
+- Type: `string[]`
+
+- Default: `[]`
+
+- Details:
+
+  Pages not included.
+
+### outFile
+
+- Type: `string`
+
+- Default: `'sitemap.xml'`
+
+- Details:
+
+  The output filename.
+
+### changefreq
+
+- Type: `'always' | 'hourly' | 'daily' | 'weekly' |'monthly' | 'yearly' | 'never'`
+
+- Default: `'daily'`
+
+- Details:
+
+  Page default update frequency.
+
+### dateFormatter
+
+- Type: `($page: Page) => string`
+
+- Required: false
+
+- Details:
+
+  Time formatter. The timestamp will be automatically generated through git plugin by default.
```

Comment:
Why a time / date formatter accepting a `page` arg ? Consider renaming it to `dateGetter` or so.

Also, it's better to provide an example.